mock/data/siste.js

```diff
@@ -0,0 +1,121 @@
+function addDaysToDate(days) {
```

Comment:
Det finnes en liten beskrivelse i README på hvordan man kan justere data for å teste lokalt som refererer til `siste.json`.
Dette bør endres til `siste.js`